fhir-install/Dockerfile

```diff
@@ -41,32 +43,20 @@ COPY --chown=1001:0 --from=base /opt/ibm-fhir-server /opt/ibm-fhir-server
 
 COPY target/LICENSE /licenses/
 
-ENV FHIR /opt/ol/wlp/usr/servers/fhir-server
-
-# Set the working directory to the fhir-server liberty server
-WORKDIR ${FHIR}
-
-# Set the output directory so that output will appear under the working directory
-ENV WLP_OUTPUT_DIR=/opt/ol/wlp/usr/servers
+ENV FHIR_CONFIG_HOME=/opt/ol/wlp/usr/servers/defaultServer \
+    WLP_LOGGING_CONSOLE_SOURCE=message,trace,accessLog,ffdc,audit \
+    WLP_LOGGING_CONSOLE_LOGLEVEL=info \
+    WLP_LOGGING_CONSOLE_FORMAT=SIMPLE \
+    WLP_LOGGING_MESSAGE_SOURCE="" \
+    WLP_LOGGING_MESSAGE_FORMAT=JSON \
+    TRACE_FILE=stdout \
+    TRACE_FORMAT=BASIC
 
-# Tell liberty not to worry about a keystore since we provide our own at a different path
-ENV KEYSTORE_REQUIRED "false"
+# Set the working directory to the liberty defaultServer
+WORKDIR ${FHIR_CONFIG_HOME}
 
-# Replace links to defaultServer with links to fhir-server and 
-# create the configDropins directories for the fhir-server
-USER root
-RUN chmod -R 755 /opt/ibm-fhir-server && \
-    ln -sfn ${FHIR} /output && \
-    ln -sfn ${FHIR} /config && \
-    rm -rf /opt/ol/wlp/output && \
-    rm -rf /opt/ol/wlp/usr/servers/defaultServer && \
-    mkdir -p /config/configDropins/defaults /config/configDropins/overrides && \
-    chown -R 1001:1001 /config/configDropins/ && \
-    yum update -y && \
-    yum clean all && \
-    rm -rf /var/cache/yum
-
-USER 1001
+RUN features.sh && \
+    VERBOSE=true configure.sh
 
 ENTRYPOINT ["/opt/ibm-fhir-server/bootstrap.sh"]
-CMD ["/opt/ol/wlp/bin/server", "run", "fhir-server"]
+CMD ["/opt/ol/wlp/bin/server", "run", "defaultServer"]
```

Comment:
as I look at this... I realize that this really deserves a highlight of some sorts... maybe in the release notes or a quick video on getting started?

Reply:
we should *for sure* highlight this change in the release notes...anyone extending this image needs to know about this change